### PR TITLE
Add Getting Started section to Contributor Guidelines

### PR DESCRIPTION
**Pull request by GloverDonovan**
This change attempts to lower the barrier to entry for potential contributors.

It addresses the following:

- Finding good first issues
- Commenting and asking questions on issues
- Contributing to older issues

Would appreciate a review.

Ref: #74

### CODE REVIEW

**Comment by domfarolino** [comment]
I might change "to let others know that you want on the issue" to "to let others know you will be working on the issue", but that's a nitpick. Also on line 15 I'd change "When" to "Whether".

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ These are the general guidelines for contributing to WHATWG standards.
 
 See [COMMITTING.md](COMMITTING.md) for further details.
 
+## Getting Started
+
+If this is your first time contributing to the WHATWG standards, check out the global list of [good first issues](https://github.com/search?q=is%3Aopen+label%3A%22good+first+issue%22+user%3Awhatwg&type=Issues) and find one you want to work on. You should comment in the thread to let others know that you want on the issue. This is a great time to ask any questions that you may have.
+
+Contributions to older issues are always appreciated. If an issue hasn't been updated in a while, you should ask if the issue is still relevant before working on it. If someone else was previously working on an issue, and you want to work on it, it's polite to ask that person if you can work on the issue before taking it.
+
+When an issue is inactive or not depends on the type of issue. In general, issues that haven't been updated in 90 days are considered inactive.
```
Comment (domenic) [inline CONTRIBUTING.md]: I don't quite understand what this "inactive" concept is meant to convey.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ These are the general guidelines for contributing to WHATWG standards.
 
 See [COMMITTING.md](COMMITTING.md) for further details.
 
+## Getting Started
+
+If this is your first time contributing to the WHATWG standards, check out the global list of [good first issues](https://github.com/search?q=is%3Aopen+label%3A%22good+first+issue%22+user%3Awhatwg&type=Issues) and find one you want to work on. You should comment in the thread to let others know that you want on the issue. This is a great time to ask any questions that you may have.
```
Comment (domenic) [inline CONTRIBUTING.md]: Missing words: "want on the issue" should be "want to work on the issue"

File: CONTRIBUTING.md (hunk 1 above)
Comment (GloverDonovan) [reply CONTRIBUTING.md]: Removing this. I just added it as an option, but I don't think it helps much.

File: CONTRIBUTING.md (hunk 2 above)
Comment (GloverDonovan) [reply CONTRIBUTING.md]: Thanks, fixed.

**Comment by GloverDonovan** [comment]
@domfarolino Got rid of the last paragraph, but I used your other suggestion. Let me know what you think.

**Comment by domfarolino** [comment]
LGTM

**Pull request merged**